client/src/components/QuestionsAnswers/QuestionForm.jsx

```diff
@@ -126,7 +119,7 @@ const QuestionForm = ({ productId, length, handleNewQuestion }) => {
                   type='text'
                   placeholder='Example: jackson11!'
                   maxLength='60'
-                  onChange={handleNicknameChange}
```

Comment:
These handle changes might be able to be combined into one handle change that takes in e.target.parentNode or some similar parameter to notify the function on which set of state to change. With 3 state setter and handle change functions, I think it might be more verbose to combine these into one function, but at 5+ I think it would start to be more economical to combine these functions.